Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,14 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+function task1(){
+  let result=hogwarts.filter(ele=>ele.house==="Gryffindor")
+  return result.map(ele=>ele.firstName+" "+ele.lastName)
```

Comment:
This works, but doesn't use object destructuring - can you rewrite this so that you use object destructuring?

Reply:
sure thanks! done